doc.xml

```diff
@@ -416,10 +416,12 @@ NIL</pre>
         </clix:function>
 
         <clix:function name="with-output-to-string*" macro="true">
-          <clix:lambda-list>(&amp;key indent) &amp;body body</clix:lambda-list>
+          <clix:lambda-list>(&amp;key indent stream-symbol) &amp;body body</clix:lambda-list>
           <clix:returns>result*</clix:returns>
           <clix:description>
-            Set up a JSON streaming encoder context, then evaluate
+			  Set up a JSON streaming encoder context on 
+			  <clix:arg>stream-symbol</clix:arg> (by default 
+			  a <clix:arg>GENSYM</clix:arg>), then evaluate
```

Comment:
"GENSYM" is not an argument and should not be denoted like that.  The CLHS also spells it in lower case if used as a noun.

Reply:
Fixed in next commit.